stdlib/marshal.mli

```diff
@@ -139,10 +141,11 @@ val from_channel : in_channel -> 'a
    one of the [Marshal.to_*] functions, and reconstructs and
    returns the corresponding value.
 
-   It raises [End_of_file] if the function has already reached the
-   end of file when starting to read from the channel, and raises
-   [Failure "input_value: truncated object"] if it reaches the end
-   of file later during the unmarshalling. *)
+   @raise End_of_file if the function has already reached the end
+   of file when starting to read from [chan].
+   
+   @raise Failure if it reaches the end of the file during
+   the unmarshalling. *)
```

Comment:
```suggestion
   @raise End_of_file if [chan] is already at the end of the file.

   @raise Failure if the end of the file is reached during
   unmarshalling itself. *)
```
I suggest the re-wording only because the whitespace needed updating anyway - I think it makes the point more clearly that if you read a value from a file which is at EOF then you'll get `End_of_file` (i.e. you can read a sequence of values of from a file in the same way as you'd read a sequence of lines, terminating when you encounter `End_of_file`) but if you get `Failure` if the function reaches EOF _while_ reading a value.